wapitiCore/attack/mod_log4shell.py

```diff
@@ -98,23 +125,31 @@ async def _verify_headers_vulnerability(
     ):
         for header, payload in malicious_headers.items():
             header_uuid = headers_uuid_record.get(header)
+            await self._verify_header_vulnerability(modified_request, header, payload, header_uuid)
 
-            if await self._verify_dns(str(header_uuid)) is True:
-                await self.add_vuln_critical(
-                    category=NAME,
-                    request=modified_request,
-                    info=_("URL {0} seems vulnerable to Log4Shell attack by using the {1} {2}") \
-                        .format(modified_request.url, "header", header),
-                    parameter=f"{header}: {payload}"
-                )
-
-                log_red("---")
-                log_red(
-                    _("URL {0} seems vulnerable to Log4Shell attack by using the {1} {2}"),
-                    modified_request.url, "header", header
-                )
-                log_red(modified_request.http_repr())
-                log_red("---")
+    async def _verify_header_vulnerability(
+        self,
+        modified_request: Request,
+        header: str,
+        payload: str,
+        unique_id: uuid.UUID
+    ):
+        if await self._verify_dns(str(unique_id)) is True:
+            await self.add_vuln_critical(
+                category=NAME,
+                request=modified_request,
+                info=_("URL {0} seems vulnerable to Log4Shell attack by using the {1} {2}") \
```

Comment:
please check fr.po for translation issue

Reply:
done